modules/cxf/src/test/java/org/mule/module/cxf/CxfInboundMessageProcessorTestCase.java

```diff
@@ -77,11 +77,9 @@ public MuleEvent process(MuleEvent event) throws MuleException
         MuleEvent event = getTestEvent(msg, getTestInboundEndpoint(REQUEST_RESPONSE));
         
         MuleEvent response = processor.process(event);
-        
         Object payload = response.getMessage().getPayload();
-        assertTrue(payload instanceof OutputHandler);
         
-        ((OutputHandler) payload).write(response, System.out);
```

Comment:
why was this removed?

Reply:
We shouldn't write to sysout directly and this had no purpose. I added an assertion on the response message instead.